skyportal/facility_apis/kait.py

```diff
@@ -0,0 +1,133 @@
+import requests
+from requests.auth import HTTPBasicAuth
+
+from . import FollowUpAPI
+from baselayer.app.env import load_env
+
+from ..utils import http
+
+env, cfg = load_env()
+
+
+if cfg['app.kait.port'] is None:
+    KAIT_URL = f"{cfg['app.kait.protocol']}://{cfg['app.kait.host']}"
+else:
+    KAIT_URL = (
+        f"{cfg['app.kait.protocol']}://{cfg['app.kait.host']}:{cfg['app.kait.port']}"
+    )
+
+
+class KAITRequest:
+
+    """A dictionary structure for KAIT ToO requests."""
+
+    def _build_payload(self, request):
+        """Payload json for KAIT queue requests.
+
+        Parameters
+        ----------
+
+        request: skyportal.models.FollowupRequest
+            The request to add to the queue and the SkyPortal database.
+
+        Returns
+        ----------
+        payload: json
+            payload for requests.
+        """
+
+        for filt in request.payload["observation_choices"]:
+            assert filt in ["U", "g", "r", "i", "z", "B", "V", "R", "I"]
+
+        assert request.payload["observation_type"] in ["long", "short"]
```

Comment:
Do not use `assert`s in production code (usually only for testing). Instead raise an exception explicitly. E.g.
```python
if request.payload.get("observation_type") not in ["long", "short"]:
     raise ValueError("Improper observation_type")
```
Using the `.get` instead of indexing `[]` should be safer in case `request.payload` does not have the requested key.